wifiphisher/common/interfaces.py

```diff
@@ -366,13 +451,18 @@ def is_interface_valid(self, interface_name, mode=None):
         # raise an error if interface can't be found
         try:
             interface_adapter = self._name_to_object[interface_name]
+            if mode == "internet":
+                interface_adapter.is_internet_access = True
 
             # raise an error if interface doesn't support the mode
+            if mode != "internet" and interface_adapter.is_managed_by_nm:
```

Comment:
> Because here you are not checking if the monitor or the AP interfaces are managed.

I'm not sure I got your point. Here I check the `mode != internet and interface_adapter.is_managed_by_nm` means mode may be equal to deauth or ap and these cards are managed by NetworkManager which is a condition we need to raise exception, and this is exactly what you say, right?